_candidates_en/Yoshida_1.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Yoshida
+title: "Getting familliar with DB by leveraging ScalikeJDBC and Skinny ORM"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: grimrose
+github: grimrose
+organization: Yokohama.groovy
+tags:
+  - Tools
+  - Others
+---
+ScalikeJDBC is a powerful and flexible DB access library. In combination with Skinny ORM, we can create a better relationship with the database. I'll introduce a case-study utilizing SQL to tackle the difficulty of database and friendly CRUD API based on one year work experience.
```

Comment:
> Line 1: Should "DB" be written out as "data base"?

Yes, but as one word _database_. In general though we can assume that the audience knows what a database is and does. I think the question is _a_ vs _the_. It's not like _the_ fridge where we can assume that everyone has one in their kitchen. So it's more like _a_ database in my opinion.

> Line 2: What is "database" - does this need to be more specific?

How about: In conjunction with Skinny ORM, it provides better handle to dealing with databases.

> Line 2: "friendly" does not make sense in this context, but I don't know the subject area well enough to make an inference as to how I should edit this.

> Line 3: Is "one year work experience" referring to one year running this program?

How about: I'll share my experience of using them in production for a year: SQL provides more direct access to the database, whereas the CRUD API provides easier to use wrapper around the database.